.travis.yml

```diff
@@ -13,6 +13,7 @@ cache:
   pip:
   directories:
     - dynamodb
+    - minio
```

Comment:
I'll take a look at that. minio was super easy to get set up by just hotwiring the endpoint, as opposed to having to pre-populate some static mock data.